dm/common/util.go

```diff
@@ -0,0 +1,44 @@
+package common
+
+import (
+	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/log"
+	"github.com/pingcap/errors"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"time"
```

Comment:
Maybe you can:
1. add a license header for every newly added file
2. reorder the `import`s to 3 sections: builtin, 3th-party, and current project (the same for other updated files).